### PR TITLE
[fix] allow puma server to bind to IP in conf file

### PR DESCRIPTION
**Pull request by thifranc**
I had to use this at work as we have reverse proxies, and we do not want puma server to be available on port 8000 from everywhere for example

### CODE REVIEW

File: config/puma.rb [hunk 1]
```diff
@@ -11,7 +11,9 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 80 }
+port=ENV.fetch("PORT") { 80 } 
+host=ENV.fetch("HOST") { "0.0.0.0" }
```
Comment (lkiesow) [inline config/puma.rb:15]: Robocop complains about the missing spaces around `=` and the trailing spaces.

Comment (thifranc) [reply config/puma.rb]: Thanks for your reply, I've fixed that :) ( not used to automated checks )

**Comment by thifranc** [comment]
Hi, do i have to do something more in order to merge, it seems you've approved my fix, but from my side the merge is still blocked in a state ' Merging can be performed automatically with 1 approving review. ', so I don't know what to do next.

thks

**Comment by lkiesow** [comment]
Hi @thifranc, unfortunately, I don't have the permissions to merge this. Someone with the appropriate permissions will need to check this again and do that. Due to the ongoing work-from-home-do-lots-of-video-conferences situation, it might take a bit of time before people actually get to do all the reviews right now.

**Comment by thifranc** [comment]
Hi,

thks for your reply, no problem for waiting someone to merge this :) happy I could help !

**Comment by farhatahmad** [comment]
Is this any different than just passing `-b 0.0.0.0` to the `rails s` command? 

```
$ rails s --help
Usage:
  rails server [puma, thin etc] [options]

Options:
  -b, [--binding=IP]                       # Binds Rails to the specified IP - defaults to 'localhost' in development and '0.0.0.0' in other environments'.
```

**Review by erisd — APPROVED** [review]

**Comment by thifranc** [comment]
> Is this any different than just passing `-b 0.0.0.0` to the `rails s` command?
> 
> ```
> $ rails s --help
> Usage:
>   rails server [puma, thin etc] [options]
> 
> Options:
>   -b, [--binding=IP]                       # Binds Rails to the specified IP - defaults to 'localhost' in development and '0.0.0.0' in other environments'.
> ```
Hi,

No I don't think it's different. Moreover in this end I'm finally using this option rather thant the other ( but I can't remember why ) :

I'm setting PORT variable in the .env file, then I modify ./bin/start to get the following line :

```
exec bundle exec puma -C config/puma.rb -b "tcp://$HOST:$PORT"
```

If it's more useful or clear, I'd be glad to propose a new PR like this

**Comment by farhatahmad** [comment]
Hi,

To help us better test and review community PRs, we have now implemented a **mandatory** PR template. We would really appreciate it if you were to update the description of this PR to match the template below. 

For more information, please see: https://github.com/bigbluebutton/greenlight#source-code--contributing

The template must be in the following format (it is automatically applied to all new PRs):
```
<!--- 
IMPORTANT
This template is mandatory for all Pull Requests.
Please follow the template to ensure your Pull Request is reviewed.
-->

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Testing Steps
<!--- Please describe in detail how to test your changes. -->

## Screenshots (if appropriate):
<!--- Please include screenshots of ALL visual changes. -->

```

**Comment by farhatahmad** [comment]
This MR is missing the template. If you would like this merged, please open a new MR and follow the template provided

**Comment by thifranc** [comment]
seems to be already integrated in latest releases, I'll leave it like that :)

**Pull request closed without merging**